packages/webapp/webapp_server.js

```diff
@@ -1346,16 +1345,21 @@ async function runWebAppServer() {
     }
   });
 
+  const suppressErrors = function() {
+    suppressExpressErrors = true;
+  };
+
   // start up app
   _.extend(WebApp, {
-    expressHandlers: packageAndAppHandlers,
-    rawExpressHandlers: rawExpressHandlers,
+    connectHandlers: packageAndAppHandlers,
+    handlers: packageAndAppHandlers,
+    rawConnectHandlers: rawExpressHandlers,
+    rawHandlers: rawExpressHandlers,
     httpServer: httpServer,
     expressApp: app,
     // For testing.
-    suppressConnectErrors: function() {
-      suppressConnectErrors = true;
-    },
+    suppressConnectErrors: suppressErrors,
+    suppressExpressErrors: suppressErrors,
```

Comment:
Would it make sense to add runtime deprecation warnings?

Reply:
IMO yes. Do you agree?

Reply:
Yeah, let's do that. While they have very similar APIs, there can be some nuanced differences.

Reply:
https://github.com/meteor/meteor/blob/e0631738f2a8a914d8a50b1060e8f40cb0873680/packages/meteor/helpers.js#L211-L222

Reply:
done!